packages/icons/pom.xml

```diff
@@ -1,240 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Is the purpose of the PR?

Reply:
It's not used anymore, and with that, sourceclear detect that it's a maven project